README.md

```diff
@@ -238,23 +238,60 @@ Supported providers with their required environment variables:
-| More coming...                                               |                                                             |                                    |
+| [`amazon-s3`](https://aws.amazon.com/s3)                     | `AWS_ACCESS_KEY_ID`<br/>`AWS_SECRET_ACCESS_KEY`             | `endpoint`<br/>`bucket` (optional) | [Pricing](https://aws.amazon.com/s3/pricing/)                            |
+| More coming...                                               |                                                             |                                    |                                                                          |
+
 
 #### Provider feature set
 
-|                              | Mux (default) | Vercel Blob | Backblaze   |
-| ---------------------------- | ------------- | ----------- | ----------- |
-| Off-repo storage             | ✅            | ✅          | ✅          |
-| Delivery via CDN             | ✅            | ✅          | -           |
-| BYO player                   | ✅            | ✅          | ✅          |
-| Compressed for streaming     | ✅            | -           | -           |
-| Adapt to slow networks (HLS) | ✅            | -           | -           |
-| Automatic placeholder poster | ✅            | -           | -           |
-| Timeline hover thumbnails    | ✅            | -           | -           |
-| Stream any soure format      | ✅            | -           | -           |
-| AI captions & subtitles      | ✅            | -           | -           |
-| Video analytics              | ✅            | -           | -           |
-| Pricing                      | Minutes-based | GB-based    | GB-based    |
+|                              | Mux (default) | Vercel Blob | Backblaze | Amazon S3 |
+| ---------------------------- | ------------- | ----------- | --------- | --------- |
+| Off-repo storage             | ✅            | ✅          | ✅        | ✅        |
+| Delivery via CDN             | ✅            | ✅          | -         | -         |
+| BYO player                   | ✅            | ✅          | ✅        | ✅        |
+| Compressed for streaming     | ✅            | -           | -         | -         |
+| Adapt to slow networks (HLS) | ✅            | -           | -         | -         |
+| Automatic placeholder poster | ✅            | -           | -         | -         |
+| Timeline hover thumbnails    | ✅            | -           | -         | -         |
+| Stream any soure format      | ✅            | -           | -         | -         |
+| AI captions & subtitles      | ✅            | -           | -         | -         |
+| Video analytics              | ✅            | -           | -         | -         |
+| Pricing                      | Minutes-based | GB-based    | GB-based  | GB-based  |
+
+
+## Required Permissions for Amazon S3
```

Comment:
Wow, that's an annoying step. I'd love to have this in a different doc rather than cluttering up the readme... but we can save that for a future docs build.